pandas/io/formats/style.py

```diff
@@ -316,6 +316,10 @@ def concat(self, other: Styler) -> Styler:
             inherited from the original Styler and not ``other``.
           - hidden columns and hidden index levels will be inherited from the
             original Styler
+          - ``css`` will be inherited from the original Styler, and the value of
+            keys ``data``, ``row_heading`` and ``row`` will be prepended with
+            ``foot_``. If several stylers are chained, they will all have only one
+            ``foot_`` prepended.
```

Comment:
The docs here need to be updated for the new foot index.
Otherwise I think this is good to go afterwards.

Note to self: Add issue for Styler copying. Add issue for broader CSS documentation of styler HTML.